src/js/tracks/track-types.js

```diff
@@ -0,0 +1,44 @@
+import AudioTrackList from './audio-track-list';
+import VideoTrackList from './video-track-list';
+import TextTrackList from './text-track-list';
+import HTMLTrackElementList from './html-track-element-list';
+
+import TextTrack from './text-track';
+import AudioTrack from './audio-track';
+import VideoTrack from './video-track';
+import HTMLTrackElement from './html-track-element';
+
+import mergeOptions from '../utils/merge-options';
+
+export const NORMAL = {
+  audio: {
+    List: AudioTrackList,
+    Track: AudioTrack,
+    Capital: 'Audio'
```

Comment:
I think these should maybe be `capital:` instead of `Capital:`. I feel like, as-is, it could be confused for a constructor because of the naming convention. And maybe `capitalName` or something?